src/Components/Admin/Profile/index.jsx

```diff
@@ -105,78 +105,70 @@ const AdminProfile = () => {
         }}
       />
       <div className={styles.formContainer}>
-        <h2 className={styles.formTitleTwo}>admin data</h2>
+        <h2 className={styles.formTitleTwo}>
+          {editMode
+            ? `${admin?.firstName} ${admin?.lastName}'s Profile`
+            : `${admin?.firstName} ${admin?.lastName}'s Profile`}
```

Comment:
You should remove the ' 's ' 
```suggestion
            : `${admin?.firstName} ${admin?.lastName} Profile`}
```